src/cpu/aarch64/injectors/jit_uni_eltwise_injector.cpp

```diff
@@ -1706,7 +1706,7 @@ void jit_uni_eltwise_injector_f32<isa>::prepare_table(bool gen_table) {
         /*        for (size_t d = 0; d < len; d += sizeof(table_entry_val_t))
             h->dd(te.val);*/
         for (size_t d = 0; d < len; d += sizeof(table_entry_val_t))
-            h->dw(te.val);
+            h->dd(te.val);
```

Comment:
Hi, @kwiersch 

 I'm sorry for the confusion. Recently, we've renamed `dw` to `dd` so that it would be the same as `dd` of `Xbyak`.

||dw|dd|
|----|----|----|
|Previous Xbyak_aarch64| write 32-bit data | not implemented |
|The latest Xbyak_aarch64 (this PR)| not implemented | write 32-bit data|
|Xbyak (for x64)| write 16-bit data | write 32-bit data|

Reply:
Hi @kawakami-k , thanks for the explanation! In that case, it makes perfect sense.